gui/encyclopedia/articles/ancient_world/warfare/spearthrower.json

```diff
@@ -0,0 +1,5 @@
+{
+    "title": "The Spear-Thrower",
+    "subtitle": "The game-changing hunting weapon of the Stone Age",
+    "content": "The spear-thrower was a hunting weapon prevalent during the later Stone Age. Evidence of its use can be found all around the world - from the Arctic to the Americas to Australia. The device is therefore also frequently referred to with the native Aztec term “Atlatl”, the Aboriginal equivalent of “Woomera”, and other indigenious names.Despite its misleading English translation, the spear-thrower's projectiles are generally classified as “darts”: they were around the length of ordinary javelins, but featured large fletchings and were really thin, lightweight, and flexible.\n\nThe throwing device itself was essentially made up of a shaft (about a third of the dart's length) with a small spur at its end fitting into a notch at the dart's butt end.\n\nGenerally, the launching motion was similar to throwing a javelin, but the spear-thrower acted as an extension of the arm and was rotated along during the swing. This resulted in additional leverage and increased projectile momentum. Today, the same principle is applied for baseball, tennis ball, and even dog ball throwing arms.\n\nMoreover, the dart's low stiffness enabled it to function as a spring: Because the energy was exerted to its butt end, the dart flexed (compressed) at first and only then pushed itself forward by extending again - and taking on a characteristic wobbling motion while flying. This behavior may seem strange at first but is actually highly energy-efficient and even improves the dart's stability in the air.\n\nBecause of these facts, an average thrower could achieve double the maximum range with a spear-thrower. Which came in especially handy when hunting large animals (like mammoths), and the distance one can afford to keep from the target could be deciding over life and death.However, there were some drawbacks to these strengths: the throwing motion was somewhat unintuitive; learning to accurately aim required considerable amounts of training. Additionally, its darts took good effort to produce, weren't well portable, and (most importantly) proved useless against armored targets. For these reasons, as large-scale warfare emerged, the spear-thrower eventually gave way to the bow and arrow as well as the javelin."
```

Comment:
```suggestion
    "content": "The spear-thrower was a hunting weapon prevalent during the later Stone Age. Evidence of its use can be found all around the world - from the Arctic to the Americas to Australia. The device is therefore also frequently referred to with the native Aztec term “Atlatl”, the Aboriginal equivalent of “Woomera”, and other indigenous names. Despite its misleading English translation, the spear-thrower's projectiles are generally classified as “darts”: they were around the length of ordinary javelins, but featured large fletchings and were really thin, lightweight, and flexible.\n\nThe throwing device itself was essentially made up of a shaft (about a third of the dart's length) with a small spur at its end fitting into a notch at the dart's butt-end.\n\nGenerally, the launching motion was similar to throwing a javelin, but the spear-thrower acted as an extension of the arm and was rotated along during the swing. This resulted in additional leverage and increased projectile momentum. Today, the same principle is applied for baseball, tennis ball, and even dog ball throwing arms.\n\nMoreover, the dart's low stiffness enabled it to function as a spring: Because the energy was exerted to its butt-end, the dart flexed (compressed) at first and only then pushed itself forward by extending again - and taking on a characteristic wobbling motion while flying. This behavior may seem strange at first, but is actually highly energy-efficient and even improves the dart's stability in the air.\n\nBecause of these facts, an average thrower could achieve double the maximum range with a spear-thrower. Which came in especially handy when hunting large animals (like mammoths), and the distance one can afford to keep from the target could be deciding over life and death. However, there were some drawbacks to these strengths: the throwing motion was somewhat unintuitive; learning to accurately aim required considerable amounts of training. Additionally, its darts took good effort to produce, weren't well portable, and (most importantly) proved useless against armored targets. For these reasons, as large-scale warfare emerged, the spear-thrower eventually gave way to the bow and arrow as well as the javelin."

Reply:
Basically a couple misspellings and some spaces that weren't there.